apps/research-service/src/site-polygons/site-polygon-query.builder.ts

```diff
@@ -104,10 +104,13 @@ export class SitePolygonQueryBuilder extends PaginatedQueryBuilder<SitePolygon>
   }
 
   async addSearch(searchTerm: string) {
-    if (!searchTerm) return this;
-
     return this.where({
-      [Op.or]: [{ "$site.name$": { [Op.like]: `%${searchTerm}%` } }, { polyName: { [Op.like]: `%${searchTerm}%` } }]
+      [Op.or]: [
+        { "$site.name$": { [Op.like]: `${searchTerm}%` } },
+        { polyName: { [Op.like]: `% ${searchTerm}%` } },
+        { "$site.name$": { [Op.like]: `${searchTerm}%` } },
+        { polyName: { [Op.like]: `% ${searchTerm}%` } }
```

Comment:
The two `polyName` queries are the same, and the two site name queries are the same. I think you're looking for this: 
```
        { "$site.name$": { [Op.like]: `${searchTerm}%` } },
        { "$site.name$": { [Op.like]: `% ${searchTerm}%` } },
        { polyName: { [Op.like]: `${searchTerm}%` } },
        { polyName: { [Op.like]: `% ${searchTerm}%` } }
 ```